tasks/ldap.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Enable the LDAP plugin from command line using sudo
+  shell: "sudo -u {{ nextcloud_system_user }} php occ app:enable user_ldap"
+  args:
+    chdir: "{{ nextcloud_path }}"
+  changed_when: false
+
+- name: Get LDAP config id
+  shell: "sudo -u {{ nextcloud_system_user }} php occ ldap:show-config | grep -w 'Configuration' | awk -F'|' '{print $3}'"
+  args:
+    chdir: "{{ nextcloud_path }}"
+  register: config_id_exists_output
+  changed_when: false
+
+- name: Get config id from stdout
+  set_fact:
+    config_id: "{{ config_id_exists_output.stdout | trim  }}"
+  when: config_id_exists_output.stdout != ""
+  changed_when: false
+
+- name: Create empty LDAP config
+  shell: "sudo -u {{ nextcloud_system_user }} php occ ldap:create-empty-config -p"
```

Comment:
Why `sudo` instead of `become_user`? I thought we enabled SSH pipelining to circumvent the become ansible bug?

Reply:
because become is a shit solution and no we did it so we have no problem switching users

Reply:
Why is become a shit solution? It is intended exactly for tasks like this, becoming a specific user.